Makefile

```diff
@@ -0,0 +1,43 @@
+ALL_PY_SRCS := setup.py \
+	$(shell find tourniquet -name '*.py') \
+	$(shell find tests -name '*.py')
+
+.PHONY: all
+all:
+	@echo "Run my targets individually!"
```

Comment:
lol just kinda thinking about this, we have a Makefile which triggers pip which triggers the setup.py which triggers cmake

Reply:
Just as god intended.